identity_account_storage/Cargo.toml

```diff
@@ -26,6 +26,7 @@ parking_lot = { version = "0.12" }
 rand = { version = "0.8", default-features = false, features = ["std", "std_rng"], optional = true }
 seahash = { version = "4.1.0", default-features = false }
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
+serde_json = { version = "1.0", default-features = false, optional = true }
```

Comment:
Again, `serde_json` probably not necessary due to `ToJson`/`FromJson` exposed by `identity_core` already.